packages/ui-react/src/components/Card/Card.tsx

```diff
@@ -98,7 +98,7 @@ function Card({
     >
       {!featured && !disabled && (
         <div className={styles.titleContainer}>
-          <h3 className={classNames(styles.title, { [styles.loading]: loading })}>{title}</h3>
+          <h2 className={classNames(styles.title, { [styles.loading]: loading })}>{title}</h2>
```

Comment:
This is not correct, because the shelf header is already an h2. However on the Playlist page (grid) it should indeed be an h2. Should we make this dynamic?

And can you maybe explain a bit further what the purpose is from the ticket? "Doesn't appear like a section heading" to me wouldn't mean the h number is incorrect.

Reply:
Dynamic is the best approach. Maybe we could introduce a hidden `h2` on Playlist page (grid) to solve it easily

Reply:
I was typing out an example, but found it better to just quickly do it and made a PR to this PR: #50. You can just merge it if you agree, and let the review happen on this one.
(Sorry if I surpass someone, but I wanted to fastest route :) ).

Reply:
> And can you maybe explain a bit further what the purpose is from the ticket? "Doesn't appear like a section heading" to me wouldn't mean the h number is incorrect.

During my research, the title for example was wrapped in a `div` element. This has been fixed during the previous sprint, so that part of the description in the ticket is slightly outdated.

Reply:
😅